app/models/user.rb

```diff
@@ -0,0 +1,43 @@
+class User
+  include ActiveModel::Model
+  attr_accessor :access_token
+
+  # Get the authorized user list from S3
+  def self.get_authorized_list
+    authorized_list = nil
+    begin
+      s3 = Aws::S3::Client.new({
```

Comment:
Hey @ktp242, 

### Roles

Creating the S3 Client like this will only work in non-production....

In production we want to instantiate it without a key and secret since it's going to get its authorization/powers via the role of the server.

We do a similar thing in the [SqsClient](https://github.com/NYPL/scsbuster/blob/master/lib/sqs_client.rb#L4).
**Let me know if you want to chat about what IAM roles are.**

### Region

I feel like you can hard-wire the region here rather than reusing the `SQS_REGION` environment variable.

Reply:
Cool I can take a look on how we did that for SQS Client.

About the region, if we made the other region for SQS a env variable, I think maybe we should make it as well?